ejb3/src/main/java/org/jboss/as/ejb3/remote/protocol/versionone/XidTransactionPrepareTask.java

```diff
@@ -91,11 +91,10 @@ private int prepareTransaction() throws Throwable {
             // check the recovery store - it's possible that the "prepare" is coming in as part of recovery operation and the subordinate
             // tx may not yet be in memory, but might be in the recovery store
             final Transaction recoveredTransaction = tryRecoveryForImportedTransaction();
-            // still not found, so just return
+            // still not found
             if (recoveredTransaction == null) {
-                if (EjbLogger.EJB3_INVOCATION_LOGGER.isDebugEnabled()) {
```

Comment:
@fl4via thanks, good point. I am thinking, however, we should log both statements as errors since we are throwing XAException afterwards, so debug level would not be appropriate anyway?